modules/apps/jaggery/emm/src/emm/modules/policy.js

```diff
@@ -128,7 +128,7 @@ policyModule = function () {
             log.error("User object was not found in the session");
             throw constants.ERRORS.USER_NOT_FOUND;
         }
-        try{
+        try {
```

Comment:
We should remove these deprecated code from the code-base for better code cleanup.